dp/subsetsum.py

```diff
@@ -0,0 +1,20 @@
+def isSubsetSum (items, length, summ):
+	if summ == 0:
+		return True
+	if length == 0 and summ != 0:
+		return False
+
+	if (items[length - 1] > summ):
+		return isSubsetSum(items, length - 1, summ);
+
+	return isSubsetSum(items, length - 1, summ) or isSubsetSum(items, length - 1, summ - items[length - 1])
+
+if __name__ == '__main__':
```

Comment:
Please write the test cases in a separate file under the test directory. Check out other test cases which use the pytest to do the testing.
Pytest documentation - https://docs.pytest.org/en/latest/contents.html